src/main/java/by/exadel/internship/service/impl/FormServiceImpl.java

```diff
@@ -253,4 +211,63 @@ public void deleteById(UUID formId) {
         log.info("Successfully deleted Form with uuid: {}", formId);
     }
 
+    private void setFeedbackByUserRole(UserDTO userDTO, Form form, FeedbackRequest feedbackRequest) {
```

Comment:
I don't like this approach. A lot of IF scenarios.  Admin and tech expert looks pretty similar. Try to refactor

Reply:
we correct this in the future pr's i added task, i only moved to the bottom this method, thanks